experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -449,6 +451,18 @@ function rebaseNodeChange<TNodeChange>(
 	const baseChange = getNodeChange(baseMark);
 	const currChange = getNodeChange<TNodeChange>(currMark);
 
+	if (markEmptiesCells(baseMark) && !isMoveMark(baseMark)) {
+		return withNodeChange(
+			currMark,
+			nodeRebaser(currChange, baseChange, NodeExistenceStateChange.Deleted),
+		);
+	} else if (markFillsCells(baseMark)) {
```

Comment:
Now that I think about it, we should check `!isMoveMark(baseMark)` here as well, since in that case (`baseMark` being a move in) nothing is actually being revived. The current implementation tolerates passing `Revived` when the node was already alive, but I think we would be breaking the contract described on `NodeRebaser`.